_appmap/env.py

```diff
@@ -188,9 +188,11 @@ def _configure_logging(self):
             loggers["appmap"]["level"] = loggers["_appmap"]["level"] = log_level
             config_dict["handlers"] = {
                 "default": {
-                    "class": "logging.FileHandler",
+                    "class": "logging.handlers.RotatingFileHandler",
                     "formatter": "default",
-                    "filename": f"appmap-{now:%Y%m%d%H%M%S}-{os.getpid()}.log",
+                    "filename": "appmap.log",
+                    "maxBytes": 5 * 1024 * 1024,
+                    "backupCount": 5,
```

Comment:
Let's make `maxBytes` bigger, say `50 * 1024 * 1024`, and set `backupCount` to `1`.